src/main/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowConstants.java

```diff
@@ -29,5 +29,6 @@ public interface WorkflowConstants {
     String CHANNEL = "Channel";
     String CHANNEL_COMP_ORIGINATEDFROM = "XOriginatedFrom";
     String CHANNEL_COMP_WEBFORM = "Webform";
-
+    String PREVIOUS_VALUE_KEY = "previous%s";
```

Comment:
```suggestion
    String PREVIOUS_VALUE_KEY = "XPrevious%s";
```

This would fit the `X...` pattern for values that should not be used outside of workflow.